aero.minova.core.application.system.service/src/main/java/aero/minova/core/application/system/ControllerExceptionHandler.java

```diff
@@ -198,6 +207,29 @@ protected SqlProcedureResult prepareExceptionReturnSqlProcedureResult(Exception
 		return result;
 	}
 
+	protected List<XSqlProcedureResult> prepareExceptionReturnXSqlProcedureResult(XProcedureException ex) {
```

Comment:
Am besten den Inhalt bzw. das Format der Liste dokumentieren:
Fehlerfreie Ergebnisse, Fehler, Leere Ergebnisse

Reply:
Das Format war in der API-Doku auch schon dokumentiert, aber ich hab sie  jetzt auch in den Code rein.